src/main/host/thread_ptrace.c

```diff
@@ -142,7 +142,7 @@ static StopReason _getStopReason(int wstatus) {
             .type = STOPREASON_EXITED_NORMAL,
             .exited_normal.exit_code = WEXITSTATUS(wstatus),
         };
-        debug("STOPREASON_EXITED_NORMAL: %d", rv.exited_normal.exit_code);
+        warning("STOPREASON_EXITED_NORMAL: %d", rv.exited_normal.exit_code);
```

Comment:
Why is it a warning if the process exited normally?

Reply:
Catching this stop-reason reaps the child, which means reaping the child will fail in Process. Instead we want to detach on the ptrace exit event and let Process get the "normal" exit.

We already have a warning in `_threadptrace_updateChildState` though, so I reverted it. I'd bumped it up to a warning while debugging.